orchestra/frontend/src/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```

Comment:
clean up?

Reply:
Err what are we looking to clean up here?